internal/commands/reminder.go

```diff
@@ -68,41 +77,44 @@ func requestStatus(ctx context.Context, client bot.Client, logger log.Logger, re
 			logger.Error(
 				ctx,
 				"command/reminder.requestStatus convertTimestamp error",
-				log.NewValue("channelID", channelID),
+				log.NewValue("channelID", incident.ChannelId),
 				log.NewValue("error", err),
 			)
 			return
 		}
 
-		if timeMessage.Before(time.Now().Add(-recurrence)) {
-			err := postMessage(client, channelID, "Update the status of this incident, just pin a message with status on the channel.")
+		if timeMessage.Before(time.Now().Add(-setRecurrence(incident))) {
+			err := postMessage(client, incident.ChannelId, statusNotify(incident))
+
```

Comment:
The reminder for a resolved incident is still related with the last pinned message on the channel. Is it make sense?
If an incident was resolved yesterday 13:00, today at 10:00 I send a message saying everything is ok and the incident is done, and I pin the message. It seems that Hellper will not notify with the resolve message.

Reply:
This is an excellent observation.